CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,485 @@
+      "items": [
+        {
+            "smartContractInfo": {
+                "type": "object",
+                "properties": {
+                    "era": {
+                        "type": "string",
+                        "description": "Describe which era this script was written for."
+
+                    },
+                    "compiler": {
+                        "type": "string",
+                        "description": "Compiler used for this script."
+                    },
+                    "compilerVersion": {
+                        "type": "string",
+                        "description": "Compiler version used for this script."
+                    },
+                    "optimizer": {
+                        "type": "string",
+                        "description": "Post compilation optimizer used for this script."
+                    },
+                    "optimizerVersion": {
+                        "type": "string",
+                        "description": "Post compilation optimizer version used for this script."
+                    },
+                    "progLang": {
+                        "type": "string",
+                        "description": "Programming language used for this script"
+                    },
+                    "repository": {
+                        "type": "string",
+                        "description": "URL to the repository of the script"
+                    }
+                }
+            },
+            "fullScriptHash": {
+              "type": "string",
+              "description": "Prefix and script hash or script hash+staking key"
+            },
+            "scriptHash": {
+              "type": "string",
+              "description": "Script hash or script hash+staking key"
+            },
+            "contractAddress": {
+              "type": "string",
+              "description": "Script on-chain address"
+            },
+            "required": [
+                "scriptHash",
+                "contractAddress"
+              ]
+        }
+      ]
+    }
+  },
+
+  "required": [
+    "subject",
+    "schemaVersion",
+    "certificationLevel",
+    "certificateIssuer",
+    "report",
+    "summary",
+    "disclaimer",
+    "scripts"
+  ]
+}
+```
+
+### Example
+
+```json
+{
+  "subject": "d684512ccb313191dd08563fd8d737312f7f104a70d9c72018f6b0621ea738c5b8213c8365b980f2d8c48d5fbb2ec3ce642725a20351dbff9861ce9695ac5db8",
+  "schemaVersion": 1,
+  "certificationLevel": 1,
+  "certificateIssuer": {
+    "name": "Audit House LLC",
+    "logo": "https://www.example.com/media/logo.svg",
+    "social": {
+        "contact": "contact@example.com",
+        "link": "https://example.com",
+        "twitter": "twiterHandle",
+        "github": "githubHandle",
+        "website": "https://www.example.com"
+    }
+  },
+  "report": {
+    "reportURLs": [
+        "https://example.io/certificate.pdf",
+        "ipfs://bafybeiemxfal3jsgpdr4cjr3oz3evfyavhwq"
```

Comment:
I would support only web based IPFS urls but I am not an expert here... is this a valid IPFS url?

Reply:
yes, the `ipfs` protocol is recognisable & it's OK for links to have only a CID (i.e. no path): https://docs.ipfs.tech/how-to/address-ipfs-on-web/#native-urls

Reply:
resolving since no disputes that the IPFS syntax is OK.